common/src/main/java/com/revenuecat/purchases/common/attribution/AttributionNetwork.kt

```diff
@@ -0,0 +1,37 @@
+package com.revenuecat.purchases.common.attribution
+
+/**
+ * Equivalent to Purchases.AttributionNetwork. We can't remove that enum class because it's public
+ */
+@Suppress("unused", "MagicNumber")
+enum class AttributionNetwork(val serverValue: Int) {
+    /**
+     * [https://www.adjust.com/]
+     */
+    ADJUST(1),
+
+    /**
+     * [https://www.appsflyer.com/]
+     */
+    APPSFLYER(2),
+
+    /**
+     * [http://branch.io/]
+     */
+    BRANCH(3),
+
+    /**
+     * [http://tenjin.io/]
+     */
+    TENJIN(4),
+
+    /**
+     * [https://developers.facebook.com/]
+     */
+    FACEBOOK(5),
+
+    /**
+     * [https://www.mparticle.com/]
+     */
+    MPARTICLE(6)
+}
```

Comment:
I have honestly thought about just deprecating it with an ERROR level and offering the replacement. People will get compilation errors but also a suggestion of how to replace it.

Reply:
Actually, I was trying deprecating this again and I don't think there is any advantage on doing so. We would still need the original `Purchases.AttributionNetwork` and the new `AttributionNetwork` in the public module. We can't just move the class (or could we), because people would get compilation errors. We are deprecating the `addAttributionData` functions anyhow, so I think the way we have it right now in this PR is fine.